examples/chat-online-status/react/tailwind.config.ts

```diff
@@ -1,28 +1,9 @@
-import type { Config } from "tailwindcss";
-import franken from 'franken-ui/shadcn-ui/preset-quick';
+import baseConfig from '../../tailwind.config';
+import type { Config } from 'tailwindcss';
 
 const config: Config = {
-  presets: [franken()],
-  content: [
-    "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
-    "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
-    "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
-  ],
-  safelist: [
-    {
-      pattern: /^uk-/,
-    },
-    "ProseMirror",
-    "ProseMirror-focused",
-    "tiptap",
-    "mr-2",
-    "mt-2",
-    "opacity-50",
-  ],
-  theme: {
-    extend: {},
-  },
-  plugins: [],
+  ...baseConfig,
+  content: [...(Array.isArray(baseConfig.content) ? baseConfig.content : []), './src/**/*.ts'],
```

Comment:
This makes sense if the last member of this array was different, but `'./src/**/*.ts'` is already in `baseConfig`, so you can take this out